### PR TITLE
Adds Incapsula to security.yml

### PR DESCRIPTION
**Pull request by ZigZagIL**

### CODE REVIEW

**Comment by mxxcon** [comment]
Your `doc:` link is to incapsula's own 2fa product. Does incapsula provide 2fa when logging in to its account?

**Comment by ZigZagIL** [comment]
Of course.

**Comment by mxxcon** [comment]
Is there a link that documents how Incapsula's users can enable 2fa on their accounts? That's what should be in `doc:` field..

**Comment by ZigZagIL** [comment]
There ya go
https://docs.incapsula.com/Content/Management%20Console%20and%20Settings/accountsettings.htm?Highlight=Factor

File: _data/security.yml [hunk 1]
```diff
@@ -49,6 +49,16 @@ websites:
       tfa: No
       twitter: gogetssl
 
+    - name: Imperva Incapsula
+      url: http://www.Incapsula.com/
```
Comment (jamcat22) [inline _data/security.yml]: Change this line to:
`url: https://www.incapsula.com`

File: _data/security.yml [hunk 2]
```diff
@@ -49,6 +49,16 @@ websites:
...
+      img: incapsula.png
+      tfa: Yes
+      email: Yes
+      sms: Yes
+      software: Yes
+      doc: https://www.incapsula.com/website-security/two-factor-authentication.html
```
Comment (jamcat22) [inline _data/security.yml]: Change this line to:
`doc: https://docs.incapsula.com/Content/Management%20Console%20and%20Settings/accountsettings.htm?Highlight=Factor`

File: _data/security.yml [hunk 3]
```diff
@@ -49,6 +49,16 @@ websites:
...
+      twitter: Incapsula_com
```
Comment (jamcat22) [inline _data/security.yml]: Remove the `twitter:` line.

**Comment by jamcat22** [comment]
@ZigZagIL Could you please make the following changes to your PR by updating your `master` branch?
- Change the `doc:` link to https://docs.incapsula.com/Content/Management%20Console%20and%20Settings/accountsettings.htm?Highlight=Factor
- Remove the Twitter handle (it's not needed for websites that have 2FA)
- Change the `url:` to be HTTPS, and all lowercase

I have also commented on the "Files changed" area with what each line should be.

Also, you may want to use a branch other then `master` in the future for PRs.

**Comment by jamcat22** [comment]
Thanks @ZigZagIL! Another contributor will review your PR before it's merged.

If it's merged, it can take awhile before Incapsula shows up on the list. You can find out more on [our wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ_MERGE).

**Pull request merged**